modules/ROOT/pages/until-successful-scope.adoc

```diff
@@ -18,35 +18,61 @@ Common processes that use Until Successful include:
+|Milliseconds Between Retries (`millisBetweenRetries`) |Specifies, in milliseconds, the minimum interval between two retries. The actual interval depends on the previous execution, but it should not exceed twice this number. The default value is 60000 milliseconds (one minute). This attribute can be either a number or an expression that resolves to a number.
 |===
 
-This XML example allows for up to five retries each second, the default. Note
-that `doc:id` attributes are removed to make the example more readable.
+== Example Configuration of the Until Successful Scope
+
+The following XML example configures a flow triggered by a Scheduler component and an Until Successful scope that executes an FTP Write operation:
 
+[source, xml, linenums]
 ----
-<flow name="untilsuccessfulFlow">
+<!-- FTP Connector config-->
+<ftp:config name="FTP_Config" doc:name="FTP Config" >
+  <ftp:connection workingDir="${ftp.dir}" host="${ftp.host}" />
+</ftp:config>
+
+<flow name="untilSuccessfulFlow" >
+  <!-- Scheduler component to trigger the flow-->
   <scheduler doc:name="Scheduler" >
     <scheduling-strategy >
-      <fixed-frequency frequency="30" timeUnit="SECONDS"/>
+      <fixed-frequency frequency="15" timeUnit="SECONDS"/>
     </scheduling-strategy>
   </scheduler>
-  <until-successful maxRetries="5" doc:name="Until Successful">
-    <!-- A single processor here -->
-    <http:request method="GET" doc:name="Request"
-      config-ref="HTTP_Request_configuration"
-      url="https://jsonplaceholder.typicode.com/users"/>
+  <!-- Until Successful scope-->
+  <until-successful maxRetries="5" doc:name="Until Successful" millisBetweenRetries="3000">
+    <!-- FTP Write operation that executes as part of the Until Successful Scope -->
+    <ftp:write doc:name="Write" config-ref="FTP_Config" path="/"/>
   </until-successful>
-  <logger level="INFO" doc:name="Logger" />
+  <logger level="INFO" doc:name="File upload success" message="File upload success"/>
+  <!-- Error Handler at flow level-->
+  <error-handler>
+    <on-error-continue enableNotifications="true" logException="true" doc:name="On Error Continue" type="RETRY_EXHAUSTED">
+      <logger level="INFO" doc:name="File upload failed" message="File upload failed"/>
+    </on-error-continue>
+  </error-handler>
 </flow>
 ----
 
+Behaviour of the example application:
+
+* If the FTP Write operation fails
++
+The Until Successful scope retries the operation every `3000` milliseconds until the operation succeeds, with a limit of `5` retries. If the last execution fails, the scope throws a `MULE:RETRY_EXHAUSTED` error, the `<on-error-continue>` handles the error and then executes the Logger with the message: `File upload failed`.
```

Comment:
The sentence reads funny. Suggestion - split the sentence in two: " If the last execution fails, the scope throws a `MULE:RETRY_EXHAUSTED` error. Then, the `<on-error-continue>` handles the error and executes the Logger with the message: `File upload failed`.